diamond/bin/diamond

```diff
@@ -23,8 +23,14 @@ import traceback
 import string
 
 import gi
+
 gi.require_version('Gtk', '3.0')
-gi.require_version('GtkSource', '3.0')
+
+try:
+  gi.require_version('GtkSource', '4')
+except:
```

Comment:
This should be `except ValueError`, but I am probably being fussy here. :)

Reply:
Good spot - fixed :)